src/Drivers/DPDK/DpdkDriverImpl.h

```diff
@@ -169,12 +163,21 @@ class DpdkDriver::Impl {
     static void txBurstErrorCallback(struct rte_mbuf* pkts[], uint16_t unsent,
                                      void* userdata);
 
+    /// Name of the Linux network interface to be used by DPDK.
+    std::string ifname;
+
     /// Stores the NIC's physical port id addressed by the instantiated
     /// driver.
-    const uint16_t port;
+    uint16_t port;
```

Comment:
I find it awkward to declare variables const and then use const_cast to bypass the access; if you prefer it that way, I can make the change.

Reply:
I agree that `const_cast` is generally a bad idea. Rather than cast away `const`, my preference is to find a way to set these variables at construction time. This typically involves the use of some helper function.